.github/workflows/cross-compiles.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             tests: none
           }
         ]
-    runs-on: ubuntu-latest
```

Comment:
Why is this only a problem on 1.1.1? (What is the reason for the change).

Reply:
the used gcc-11 version creates false-positive warnings in m68k-linux-gnu which make the build fail,
the warning is due to a large refactoring in the gcc-internal variable range computation engine ("ranger")
https://gcc.gnu.org/wiki/AndrewMacLeod/Ranger
that was rolled out in gcc-11 and gcc is still recovering from this major change.
So instead of disabling a potentially useful warning, I wanted to keep the test using ubuntu-20.04, and if that
version is no longer available, skip directly to ubuntu-24.04, which will be using gcc-13, and that warning
is already gone in gcc-trunk of today, I double-checked that.